Rocket.Chat.iOS/ViewControllers/ViewController.swift

```diff
@@ -21,20 +22,30 @@ class ViewController: UIViewController, UITableViewDataSource, UITableViewDelega
     
     // indexPath to find the bottom of the tableview
     var bottomIndexPath:NSIndexPath = NSIndexPath()
-  
-    var messages: NSArray?
     
     var dateFormatter = NSDateFormatter()
     
-    
     var meteor: MeteorClient!
     
+    //JSON to keep the response
+    var chatMessages:JSON = []
+    
+    //Dictionary to keep the users from the JSON response
+    var users = [Int():[String(),String(),String(),String()]]
```

Comment:
I am using these structure to sort the messages in a reverse order (from oldest to newest) so I can access them easier using the indexPath when the tableview is being populated. First I need to get the response as JSON and then sort it inside this users dictionary. Maybe I could comment this better so it can be clear to others? 
Something like:
[index: [id,username,message,msgType]]